docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,442 @@
+We migrated the dataset to Fairlearn after it was phased out of scikit-learn in June 2020. 
+The dataset remains in Fairlearn as an example of how systemic racism can occur in data and to 
+show the effect of Fairlearn's assessment and pre-/in-/post-processing tools on real, problematic data. 
+
+We also think this dataset provides an interesting case study of how fairness is fundamentally a
+socio-technical issue. 
+This article has the following goals:
+  * Educate users about the history of the dataset and its fairness-related harms
+  * Show users how fairness-related harms manifest in the data and in downstream modelling tasks
+  * Suggest best practices for dealing with the Boston Housing data and alternative benchmarking datasets
+
+
+.. _boston_dataset_origin:
+
+Dataset Origin and Use
+^^^^^^^^^^^^^^^^^^^^^^
+
+Contrary to expectation, the Boston Housing dataset was not developed for economic purposes.
+Harrison and Rubenfield (1978)_ 
+developed the dataset to illustrate the issues with using housing market data 
+to measure consumer willingness to pay for clean air. 
+The authors use a hedonic pricing [#9] 
+approach, which assumes that the price of a good or service can be modeled as a 
+function of features both internal and external to the good or service. 
+The input to this model was a dataset comprising the Boston Standard Metropolitan 
+Statistical Area [#10]_, with the nitric oxides concentration (*NOX*) 
+serving as a proxy for air quality.
+
+The paper sought to estimate the median value of owner-occupied homes (now 
+*MEDV*), and included the remaining variables to capture other neighborhood 
+characteristics.
+Further, the authors took the derivative of their housing 
+value equation with respect to nitric oxides concentration 
+to measure the "amount of money households were willing to pay 
+with respect to air pollution levels in their census tracts." 
+The variables in the dataset were collected in the early 1970s 
+and come from a mixture of surveys, administrative records, and other research
+papers. 
+While the paper does define each variable and suggest its impact on 
+the housing value equation, it lacks reasoning for including that particular
+set of variables.
+
+Modern machine learning practitioners have used the Boston Housing dataset as 
+a benchmark to assess the performance of emerging supervised learning 
+techniques. 
+It's featured in `scipy lectures <https://scipy-lectures.org/packages/scikit-learn/auto_examples/plot_boston_prediction.html>`_, 
+indexed in the `University of California-Irvine Machine Learning Repository 
+<https://archive.ics.uci.edu/ml/machine-learning-databases/housing/>`_ and in 
+Carnegie Mellon University's `StatLib <http://lib.stat.cmu.edu/datasets/boston>`_, 
+and for a time was included as one of scikit-learn's and tensorflow's standard toy datasets
+(see :func:`tf.keras.datasets.boston_housing`). 
```

Comment:
Yep, will do that @hildeweerts

Reply:
@hildeweerts @romanlutz So I think this is the correct way to refer to the Tensorflow API: https://github.com/GPflow/tensorflow-intersphinx/. I committed an updated version of the _conf_ file to this branch, so please let me know if this will work!